core/sms_server_v2.py

```
@@ -0,0 +1,592 @@
+    response_model=OnboardRegisterResponse,
+    responses={
+        400: {"model": ErrorResponse},
+        403: {"model": ErrorResponse},
+        429: {"model": ErrorResponse},
+    },
+    tags=["Onboarding"],
+    summary="Register mobile for onboarding",
+)
+def register_onboarding(
+    request: OnboardRegisterRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.14.10)

Unused function argument: `db`

(ARG001)